analytics/test/consumer.py

```diff
@@ -96,7 +96,7 @@ def mock_post(*args, **kwargs):
         mock_post.call_count = 0
 
         with mock.patch('analytics.consumer.post', mock.Mock(side_effect=mock_post)):
-            consumer = Consumer(None, 'testsecret')
+            consumer = Consumer(None, 'testsecret', retries=3)
```

Comment:
Is this to avoid changing the existing test?

It may be worth testing with the default value of retries (10), as that's what's most likely to be used.

Reply:
It's mostly so that we don't have to wait so long for the tests to finish. But testing with the default value is a good idea, I'll add one test to cover that and leave retries=3 for the majority of tests.